.goreleaser.yaml

```diff
@@ -106,6 +106,10 @@ dockers:
       - "--build-arg=BUILD_VERSION={{.Version}}"
       - "--build-arg=VCS_REF={{.FullCommit}}"
       - "--build-arg=VCS_URL={{.GitURL}}"
+      - "--label=io.artifacthub.package.readme-url=https://raw.githubusercontent.com/anchore/syft/main/README.md"
+      - "--label=io.artifacthub.package.logo-url=https://user-images.githubusercontent.com/5199289/136844524-1527b09f-c5cb-4aa9-be54-5aa92a6086c1.png"
+      - "--label=io.artifacthub.package.license=Apache-2.0"
+      - "--label=org.opencontainers.image.description=CLI tool and library for generating a Software Bill of Materials from container images and filesystems"
```

Comment:
Also, maybe it would be better just to link to Github for the readme so the user is presented with something nicely formatted? https://github.com/anchore/syft#readme

Reply:
Hello @kzantow, thanks for the review, but I think this README file URL is for the ArtifactHub to be able to render the README, so, it is not for the end-users of syft, am I right @dirien?

Reply:
> Very minor thing: these containers aren't really distributing libraries, just CLI tool, right? I might take out "and library".

Ofc 🙋🏻‍♂️